www/about/policies/terms-of-service.md

```diff
@@ -0,0 +1,414 @@
+     to the Service, including any representation or warranty that the use of
+the Service will (a) be timely, uninterrupted or error-free or operate in
+combination with any other hardware, software, system, or data, (b) meet your
+requirements or expectations, (c) be free from errors or that defects will be
+corrected, or (d) be free of viruses or other harmful components. Gratipay also
+makes no representations or warranties of any kind with respect to any content.
+No advice or information, whether oral or written, obtained from Gratipay or
+through the Service, will create any warranty not expressly made herein.
+
+  1. Your access to and use of the Service or any content is at your own risk.
+     YOU UNDERSTAND AND AGREE THAT THE SERVICE IS PROVIDED TO YOU ON AN "AS IS"
+AND "AS AVAILABLE" BASIS. WITHOUT LIMITING THE FOREGOING, TO THE FULL EXTENT
+PERMITTED BY LAW, GRATIPAY DISCLAIMS ALL WARRANTIES, EXPRESS OR IMPLIED, OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, OR NON-INFRINGEMENT.
+
+1. Limitation of Liability
+
+  1. TO THE MAXIMUM EXTENT PERMITTED BY APPLICABLE LAW, GRATIPAY SHALL NOT BE
+     LIABLE FOR ANY INDIRECT, INCIDENTAL, SPECIAL, CONSEQUENTIAL OR PUNITIVE
+DAMAGES, OR ANY LOSS OF PROFITS OR REVENUES, WHETHER INCURRED DIRECTLY OR
+INDIRECTLY, OR ANY LOSS OF DATA, USE, GOOD-WILL, OR OTHER INTANGIBLE LOSSES,
+RESULTING FROM (i) YOUR ACCESS TO OR USE OF OR INABILITY TO ACCESS OR USE THE
+SERVICE; (ii) ANY CONDUCT OR CONTENT OF ANY THIRD PARTY ON THE SERVICE,
+INCLUDING WITHOUT LIMITATION, ANY DEFAMATORY, OFFENSIVE OR ILLEGAL CONDUCT OF
+OTHER PARTICIPANTS OR THIRD PARTIES; (iii) ANY CONTENT OBTAINED FROM THE
+SERVICE; OR (iv) UNAUTHORIZED ACCESS, USE OR ALTERATION OF YOUR TRANSMISSIONS
+OR CONTENT.  IN NO EVENT SHALL THE AGGREGATE LIABILITY OF GRATIPAY OR ITS
+AFFILIATES EXCEED THE GREATER OF ONE HUNDRED U.S. DOLLARS (U.S. $100.00) OR THE
+AMOUNT YOU PAID GRATIPAY, IF ANY, IN THE PAST SIX MONTHS FOR THE SERVICE GIVING
+RISE TO THE CLAIM.
+
+1. Indemnification
+
+  1. You agree to defend, indemnify and hold harmless Gratipay and its
+     officers, directors, employees, contractors and agents from and against
+any and all claims, causes of action, damages, obligations, losses,
+liabilities, costs or debt, and expenses (including attorneys' fees and costs)
+and all amounts paid in settlement arising from or relating to use of the
+Service, breach of these Terms or violation of any laws. Gratipay may assume
+the exclusive defense and control of any matter for which you have agreed to
+indemnify Gratipay and you agree to assist and cooperate with Gratipay in the
+defense or settlement of any such matters.
+
+1. Miscellaneous Terms
+
+  1. Assignment: You may not assign your rights and obligations under these
+     Terms to any other person. In the event of a merger, sale, or acquisition
+of substantially all of Gratipay's assets, Gratipay may assign its rights and
+obligations under these Terms to its successor.
```

Comment:
From @copiesofcopies:

> The purpose of the asymmetry in the assignment provision is that Gratipay could be placed at risk if someone assigned their rights and obligations to, e.g., someone it was illegal for Gratipay to make payments to. But in the event Gratipay decides to restructure or be acquired, it would be a lot of trouble to request permission of every user before doing so. I think we could balance this a little bit by amending the first sentence thusly: "You may not assign your rights and obligations under these Terms to any other person **except with Gratipay's express written agreement**."